lib/brexit_checker/question.rb

```diff
@@ -44,6 +45,12 @@ def self.load(params)
 
   def self.load_all
     @load_all = nil if Rails.env.development?
-    @load_all ||= YAML.load_file(CONFIG_PATH)['questions'] .map { |q| load(q) }
+    @load_all ||= YAML.load_file(CONFIG_PATH)['questions'].map.with_index { |q, i| load(i, q) }
+  end
+
+  def self.resolve(criteria_keys: [], previous_question: 0)
```

Comment:
I'm wondering if this method really needs to be in the Question class. This is partly my allergy of putting things in models (because so much can go into them), but also that a helper method might lend itself to a more descriptive name, such as `next_question`, whereas it would be weird to say `Question.next_question`.

Reply:
Yup - moved to helper.